src/etl.py

```diff
@@ -0,0 +1,17 @@
+from pyspark.sql import SparkSession
+from prepare import Prepare
+from datastore import DataStore
+from config import app_config
+
+if __name__ == '__main__':
+    spark_ses = SparkSession \
```

Comment:
Please refactor the initialization of the SparkSession into the `AppContext` module. Add a method `init_spark` which has also access to the loaded config. Great job on passing in the entire config map - well done! 🚀